### ISSUE TITLE
3v3 queue crashing Algalon

### ISSUE THREAD

**Issue by Darthblaider** [dalaranwow/dalaran-wow#3661]
After sitting in the 3v3 arena queue for nearly two hours, we finally found another team to play against. Upon them queuing and us all attempting to accept the queue pop, the server immediately went on the fritz. It was that exact moment when the queue popped. 
Eventually, the server properly reset and we attempted once again to have a 3v3 match, thinking it was just dumb luck and bad timing. Once again, as we get matched for a game, the server broke.
If this doesn't have something to do with us queuing, nothing does.

This is why nobody does 3v3. This is why people win trade. So screw us for trying to get legit games. Good Game.

**Comment by robinsch** [comment]
@Lothloryen Is this valid? I doub that.

**Comment by Darthblaider** [comment]
It literally happened to us three times in a row. As soon as the server came back up we did it again. Crash. We only did it the third time for science and broke it once more.

**Comment by Lothloryen** [comment]
Hello there,

Nothing related to the arenas on the logs, since some players mentioned that I have been tracking some matches just to be safe - many 3v3 matches since that happened and everything is okay so, I will assume it was just a mere coincidence.

Thanks for the report.

**Comment by Dolmio1** [comment]
Happened again sunday 00:58 servertime, queued against Blaidetta and the server insta crashed as we entered the arena

**Comment by Darthblaider** [comment]
We believe we can confirm that it's the Guard's Mark horde receive in honor hold. As soon as the player with it clicks accept the server goes crazy and breaks. I was able to get inside the game before the server went nuts but the player in honor hold with the debuff had no such luck and as soon as he clicked accept everything went bad. 
On a similar note, this wouldn't be as much of an issue if there wasn't a million extra guards in Honor Hold, but not in Thrallmar. Though I'm sure we had a lot to do with that considering all the camping we used to do back in the day.

STEPS TO RECREATE:
1) have a horde player get marked in honor hold
2) have the arena queue pop(may work with rbg, we don't know)
3) The player with the guard's mark accepts and stuff goes bad.
Only tested with 3v3 queue and it's officially happened 4 times now.
